client/src/components/MoneyButton/MoneyButton.style.js

```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
+import addToBasket from "../../assets/addToBasket.png";
+
+export const Order = styled.section`
```

Comment:
Same points in this file about naming convention, font and colours

Reply:
As above, naming convention is unnecessary I think - happy to do the font/colours if there's a way of doing it with styled components?

Reply:
Font / Colours now done in the latest commit.